transactions/mint_fusd.cdc

```diff
@@ -0,0 +1,40 @@
+// This transaction mints new FUSD and increases the total supply.
+// The minted FUSD is deposited into the recipient account.
+//
+// Parameters:
+// - amount: The amount of FUSD to transfer (e.g. 10.0)
+// - to: The recipient account address.
+//
+// This transaction will fail if the authorizer does not have and FUSD.MinterProxy
+// resource. Use the setup_fusd_minter.cdc and deposit_fusd_minter.cdc transactions
+// to configure the minter proxy.
+//
+// This transaction will fail if the recipient does not have
+// an FUSD vault stored in their account. To check if an account has a vault
+// or initialize a new vault, use check_fusd_vault_setup.cdc and setup_fusd_vault.cdc
```

Comment:
seems no `check_fusd_vault_setup.cdc` and check already done as part of `setup_fusd_vault.cdc`.

Reply:
All the fusd set up scripts/transactions are copied over from the fusd repo including these comments. I can see why this causes confusion, I'll think of a way to separate these scripts from the ones we created, maybe put it in a different folder?

Reply:
i see, that's fine, maybe for now we can just update the comment to remove `check_fusd_vault_setup.cdc`.
Seems there is complications in creating subfolder which @KsenijaZivkovic @jacksonConrad  knows better on that.